src/range_proof.rs

```diff
@@ -214,17 +216,17 @@ where
     /// The prover must ensure that the commitments and witness opening data are consistent
     #[cfg(feature = "rand")]
     pub fn prove(
-        transcript_label: &'static str,
+        transcript: &mut Transcript,
```

Comment:
Now that `merlin::Transcript` is part of the public interface, we should consider re-exporting it in this crate so that users do not have to include a semi-specific version of `merlin` to use this crate.

Reply:
Great idea.